flask_iiif/cache/redis.py

```diff
@@ -25,7 +25,11 @@ def __init__(self):
         """Initialize the cache."""
         super(ImageRedisCache, self).__init__()
         redis_url = current_app.config['IIIF_CACHE_REDIS_URL']
-        self.cache = RedisCache(host=StrictRedis.from_url(redis_url))
+        prefix = current_app.config.get('IIIF_CACHE_REDIS_PREIFX', 'iiif')
```

Comment:
Can you add the new config variable to `config.py` so it will be [listed in the documentation](https://flask-iiif.readthedocs.io/en/latest/#module-flask_iiif.config)? :)
P.S. notice the [docstring at the beginning of the file](https://github.com/inveniosoftware/flask-iiif/blob/master/flask_iiif/config.py#L10) for the docs

Reply:
:+1: